e2e_tests/docker/indexer/Dockerfile

```diff
@@ -13,14 +13,17 @@ RUN mkdir -p /opt/go/indexer
 COPY . /opt/go/indexer
 WORKDIR /opt/go/indexer
 RUN rm -f $HOME/go/bin/algod
-RUN rm /opt/go/indexer/cmd/algorand-indexer/algorand-indexer
+RUN rm -f /opt/go/indexer/cmd/algorand-indexer/algorand-indexer
 WORKDIR /opt/go/indexer/third_party/go-algorand
 RUN make install
 WORKDIR /opt/go/indexer
 RUN make
 ENV PATH="${HOME}/go/bin/:${PATH}"
-RUN pip3 install -r misc/requirements.txt
+COPY ./e2e_tests ./e2e_tests
```

Comment:
On line 13 we have `COPY . /opt/go/indexer`. So this directory could already be available at `/opt/go/indexer/e2e_tests`